spec/views/devise/mailer/confirmation_instructions.html.slim_spec.rb

```diff
@@ -29,10 +26,8 @@
   end
 
   it 'includes a request to not reply to this messsage' do
-    assign(:resource, build_stubbed(:user, confirmed_at: Time.zone.now))
-    render
-
-    expect(rendered).to have_content 'PLEASE DO NOT REPLY TO THIS MESSAGE'
+    expect(last_email.parts.first.body.raw_source).
+      to have_content 'Please do not reply to this message.'
```

Comment:
If you run this test on its own, it will fail because there is no `last_email` variable in a view spec. No emails are being sent in view specs. I think the only change that needs to be made here is the expected text.